README.md

```diff
@@ -12,6 +12,26 @@ This project is a _fork_ of the [original QuickJS project] by Fabrice Bellard an
 Head over to the [project website] for instructions on how to get started and more
 documentation.
 
+## Download
+
+We don't do canary releases. If you want to download the latest release, GH doesn't make it very obvious, but here is how: 
```

Comment:
I'd rather see this here: https://github.com/quickjs-ng/quickjs/blob/master/docs/docs/installation.md and a link to it in the README.

Reply:
The idea is to make getting the latest QuickJS NG conspicuous. Without having to navigate away from the README to get instructions about how to do that.

Reply:
I don't want to have duplicated information. It's just one click away.